src/strategies/etherorcs-combo-balanceof/schema.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Strategy",
+  "definitions": {
+    "Strategy": {
+      "title": "Strategy",
+      "type": "object",
+      "properties": { },
+      "required": ["address"],
```

Comment:
```suggestion
```

Reply:
@tempest-sol Make sure to run the test cases in your local

Reply:
I removed the schema since it's optional